compiler-rt/lib/asan/asan_interceptors.cpp

```diff
@@ -56,6 +56,7 @@ namespace __asan {
 #  define ASAN_READ_STRING(ctx, s, n) \
     ASAN_READ_STRING_OF_LEN((ctx), (s), internal_strlen(s), (n))
 
+#  if SANITIZER_INTERCEPT_STRCAT || SANITIZER_INTERCEPT_STRCPY
```

Comment:
convention is to use SANITIZER_INTERCEPT_ to controll sanitizer_common interceptors
for asan, please use ASAN_INTERCEPT_